### DISCUSSION TITLE
Scatter Plot ketika disorot tidak memunculkan informasi interactive

### DISCUSSION THREAD

**Discussion by Argo2226** [teamalgoritma/community#724] [Q&A] [answered]
Selamat pagi, mohon izin bertanya saya ingin membuat scatter plot yang ketika disorot  memunculkan informasi interaktif. Namun hasil berikut untuk setiap pointnya saya sorot tidak bisa memunculkan informasi interaktif, mohon bantuan untuk solusinya. Terima kasih banyak

Code
![image](https://github.com/teamalgoritma/community/assets/75960751/9ba1429e-6629-42ad-85f6-ac129b12e270)

Hasil
![image](https://github.com/teamalgoritma/community/assets/75960751/90bc1049-6643-4441-955d-d6f6fab1c4c4)

**Comment by arkanayudhistira** [comment, marked as answer]
Dear @Argo2226,

Berdasarkan code yang telah dikirimkan, kemungkinan penyebab munculnya tooltip interaktif adalah karena setelah dilakukan grouping menggunakan fungsi `group_by()`, tidak dilanjutkan dengan `summarise()` untuk menghitung hasil agregasi dari data berdasarkan provinsi tersebut. Silahkan dicoba untuk menambahkan `summarise()` berdasarkan data yang diinginkan.

Semoga membantu.

**Reply by Argo2226** [reply]
Siapp, saya coba lebih dahulu. Terima kasih atas sarannya
